src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/DatePicker/DatePickerSample.xaml

```diff
@@ -14,7 +18,8 @@
 		<DatePicker Grid.Row="1"
 		            Margin="15,10"
 		            HorizontalAlignment="Center"
-		            VerticalAlignment="Top">
+		            VerticalAlignment="Top"
+					Date="{Binding Date}">
 		</DatePicker>
 
 		<!-- DatePicker is broken: https://github.com/unoplatform/uno/issues/188 -->
```

Comment:
This is probably no longer applicable

Reply:
@MartinZikmund I don't understand this one. It's bound to the VM...  it was just not working in the past.